### PR TITLE
Refactor: clouseau start

### PR DESCRIPTION
**Pull request by till**
- removes maven as a dependency to run clouseau and use the compiled artefact
 - clean up log output from clouseau (via proper conf)
 - use multi-stage to further trim the final image (1.1 GB to 550 MB)

### CODE REVIEW

File: clouseau/Dockerfile [hunk 1]
```diff
@@ -52,13 +52,41 @@ RUN git clone -b ntr_master https://github.com/neutrinity/clouseau . \
...
+
+RUN apt-get -qq update -y \
+  && DEBIAN_FRONTEND=noninteractive apt-get -qq install -y apt-utils \
+  && DEBIAN_FRONTEND=noninteractive apt-get -qq install -y --no-install-recommends \
+  libnspr4 libnspr4-0d \
+  openssl \
+  curl \
+  ca-certificates \
+  unzip \
+  openjdk-7-jdk \
```
Comment (till) [inline clouseau/Dockerfile:68]: I will replace this with JRE later (should get us another win on size), and also trim other deps from the final image (like, `ca-certificates` etc.).

File: clouseau/log4j.properties [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (till) [inline clouseau/log4j.properties:1]: I borrowed this file from another Java project on Github (don't remember which). But let's use this as a base for now.

Comment (till) [reply clouseau/log4j.properties:1]: Specifically, I want to use something like this (later): https://stackoverflow.com/a/10314912

File: docker-compose-hub.yaml [hunk 3]
```diff
@@ -21,7 +21,7 @@ services:
       - ERLANG_COOKIE=cookie_monster
       - COUCHDB_USER=jan
       - COUCHDB_PASSWORD=password
-      - CLOUSEAU_NAME=clouseau@clouseau.local
+      - CLOUSEAU_NAME=clouseau.local
```
Comment (till) [inline docker-compose-hub.yaml:24]: Ignore changes to the compose files. Just had to make them to get it running, but they will be gone with #20 anyway.

**Review by lucidNTR — APPROVED** [review]
cool looks like we are not far from the first goal .-)

**Pull request merged**